test/docker/cmake-3.25.2-SHA-256.txt

```diff
@@ -0,0 +1,17 @@
+82ef7846809a6cfe282a6196b300b081ff7878e40065c1d530352f8f7a2a0c2d  cmake-3.25.2-files-v1.json
```

Comment:
do we need this?

Reply:
yes we do

Reply:
Yeah, I want to check sums, I'm paranoid like this :)